### PR TITLE
OBTW-62 initialization web api

### PR DESCRIPTION
**Pull request by Evgentus0**

### CODE REVIEW

File: src/Grouper.Api.Data/Context/DbInitializer.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+using System.Threading.Tasks;
+
+namespace Grouper.Api.Data.Context
+{
+    public class DbInitializer
+    {
+        public static void Initialize(GrouperDbContext context)
+        {
+            var notExist = context.Database.EnsureCreated();
+            var alreadyExis = !notExist;
```
Comment (vinnik-dmitry07) [inline src/Grouper.Api.Data/Context/DbInitializer.cs:14]: Ммм

File: src/Grouper.Api.Web/Models/UserModel.cs [hunk 2]
```diff
@@ -7,13 +7,13 @@ namespace Grouper.Api.Web.Models
 {
     public class UserModel
     {
-        public int Id {get; set;}
-        public string Name {get; set;}
+        public string Id {get; set;}
+        public string FirstName {get; set;}
+        public string LastName { get; set; }
         public string Login {get; set;}
```
Comment (vinnik-dmitry07) [inline src/Grouper.Api.Web/Models/UserModel.cs:13]: Логин же убрали

File: src/Grouper.Api.Data/Repositories/GroupRepository.cs [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        public async Task Create(Group group)
+        {
+            await _context.Groups.AddAsync(group);
+        }
+
+        public async Task Delete(int id)
+        {
+            var group = await _context.Groups.FirstOrDefaultAsync(x => x.Id == id);
+
+            if(group != null)
```
Comment (vinnik-dmitry07) [inline src/Grouper.Api.Data/Repositories/GroupRepository.cs:37]: Пробел

File: src/Grouper.Api.Data/Context/DbInitializer.cs (hunk 1 above)
Comment (Evgentus0) [reply src/Grouper.Api.Data/Context/DbInitializer.cs:14]: for better understanding, negative checks more harder to understand (read  clean code:))

File: src/Grouper.Api.Data/Repositories/PostRepository.cs [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+        public async Task Create(Post post)
+        {
+            await _context.Posts.AddAsync(post);
+        }
+
+        public async Task Delete(int id)
+        {
+            var post = await _context.Posts.FirstOrDefaultAsync(x => x.Id == id);
+
+            if(post != null)
```
Comment (vinnik-dmitry07) [inline src/Grouper.Api.Data/Repositories/PostRepository.cs:36]: Пробел

File: src/Grouper.Api.Infrastructure/DTOs/GroupDto.cs [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+using System.Threading.Tasks;
+
+namespace Grouper.Api.Infrastructure.DTOs
+{
+    public class GroupDto
+    {
+        public int Id { get; set; }
+        public string Name { get; set; }
+        public GroupDto ParentGroup { get; set; }
+        public List<UserDto> Pupils { get; set; }
```
Comment (vinnik-dmitry07) [inline src/Grouper.Api.Infrastructure/DTOs/GroupDto.cs:14]: Students

File: src/Grouper.Api.Web/Controllers/FormController.cs [hunk 6]
```diff
@@ -13,41 +16,63 @@ namespace Grouper.Api.Web.Controllers
...
+            List<FormDto> forms = await _formService.GetByUserId(userId);
+            var result = _mapper.Map<List<FormDto>>(forms);
 
-            return Ok(lf);
+            return Ok(result);
         }
 
         [HttpPost]
-        public async Task<ActionResult<ResponseModel>> Create()
+        public async Task<ActionResult<ResponseModel>> Create([FromBody]FormModel formModel)
```
Comment (vinnik-dmitry07) [inline src/Grouper.Api.Web/Controllers/FormController.cs:48]: Пробел

File: src/Grouper.Api.Web/Controllers/GroupController.cs [hunk 7]
```diff
@@ -13,59 +16,82 @@ namespace Grouper.Api.Web.Controllers
...
 
-            return Ok(resp);
+            var response = new ResponseModel { Message = "Group was created" };
+            return Ok(response);
         }
         
         [HttpPost]
         [Route("{groupId}/add-user")]
-        public async Task<ActionResult<ResponseModel>> AddUser(int groupId, [FromQuery]int userId)
+        public async Task<ActionResult<ResponseModel>> AddUser(int groupId, [FromQuery]string userId)
```
Comment (vinnik-dmitry07) [inline src/Grouper.Api.Web/Controllers/GroupController.cs:58]: Пробел

File: src/Grouper.Api.Web/Controllers/PostController.cs [hunk 8]
```diff
@@ -13,51 +16,74 @@ namespace Grouper.Api.Web.Controllers
...
+            var result = _mapper.Map<PostModel>(postDto);
+
+            return Ok(result);
         }
 
         [HttpPost]
-        [Route("{postId}/add-comment")]
-        public async Task<ActionResult<ResponseModel>> AddComment(int postId, [FromBody]CommentModel comment)
+        [Route("add-comment")]
+        public async Task<ActionResult<ResponseModel>> AddComment([FromBody]CommentModel comment)
```
Comment (vinnik-dmitry07) [inline src/Grouper.Api.Web/Controllers/PostController.cs:59]: Пробел

File: src/Grouper.Api.Data/Context/DbInitializer.cs (hunk 1 above)
Comment (vinnik-dmitry07) [reply src/Grouper.Api.Data/Context/DbInitializer.cs:14]: `var alreadyExist = !context.Database.EnsureCreated();`

**Pull request merged**